### PR TITLE
graphs added

### PR DESCRIPTION
**Pull request by sameerddr**

### CODE REVIEW

File: src/Components/Graphs/Graph1.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import axios from "axios";
+import { Column } from "@ant-design/plots";
+
+import "./Graph1.css";
+
+function Graph1() {
+  const [data, setdata] = useState([]);
+
+  const GetApi = async () => {
+    const response = await axios.get(
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph1.jsx]: store url in a variable/state

File: src/Components/Graphs/Graph1.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        });
+        a.push({
+          value: data[Object.keys(data)[i]]["2. high"],
+          type: "high",
+          date: Object.keys(data)[i],
+        });
+      }
+      i++;
+    }
+    console.log(a);
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph1.jsx]: remove/comment console statements before import

File: src/Components/Graphs/Graph2.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import axios from "axios";
+import { Line } from "@ant-design/plots";
+
+import "./Graph2.css";
+
+const DemoLine = () => {
+  const [data, setdata] = useState([]);
+
+  const GetApi = async () => {
+    const response = await axios.get(
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph2.jsx]: same as above

File: src/Components/Graphs/Graph2.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=SHOP.TRT&outputsize=full&apikey=W9IPCZT6X76UJZOY"
+    );
+    console.log(response);
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph2.jsx]: same as above

File: src/Components/Graphs/Graph2.jsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        });
+        a.push({
+          value: data[Object.keys(data)[i]]["2. high"],
+          type: "high",
+          date: Object.keys(data)[i],
+        });
+      }
+      i++;
+    }
+    console.log(a);
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph2.jsx]: same as above

File: src/Components/Graphs/Graph3.jsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import axios from "axios";
+import { Area } from "@ant-design/plots";
+
+import "./Graph3.css";
+
+const DemoArea = () => {
+  const [data, setdata] = useState([]);
+
+  const GetApi = async () => {
+    const response = await axios.get(
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph3.jsx]: same as above

File: src/Components/Graphs/Graph3.jsx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+      "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=RELIANCE.BSE&outputsize=full&apikey=W9IPCZT6X76UJZOY"
+    );
+    console.log(response);
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph3.jsx]: same as above

File: src/Components/Graphs/Graph3.jsx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        });
+        a.push({
+          value: data[Object.keys(data)[i]]["2. high"],
+          type: "high",
+          date: Object.keys(data)[i],
+        });
+      }
+      i++;
+    }
+    console.log(a);
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph3.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Graphs/Graph1.jsx (hunk 1 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph1.jsx]: Solved

Comment (sameerddr) [reply src/Components/Graphs/Graph1.jsx]: Solved

Comment (sameerddr) [reply src/Components/Graphs/Graph1.jsx]: Solved

File: src/Components/Graphs/Graph1.jsx (hunk 2 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph1.jsx]: Solved

File: src/Components/Graphs/Graph2.jsx (hunk 3 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph2.jsx]: Solved

File: src/Components/Graphs/Graph2.jsx (hunk 4 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph2.jsx]: Solved

File: src/Components/Graphs/Graph2.jsx (hunk 5 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph2.jsx]: Solved

File: src/Components/Graphs/Graph3.jsx (hunk 6 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph3.jsx]: Solved

File: src/Components/Graphs/Graph3.jsx (hunk 7 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph3.jsx]: Solved

File: src/Components/Graphs/Graph3.jsx (hunk 8 above)
Comment (sameerddr) [reply src/Components/Graphs/Graph3.jsx]: Solved

File: src/Components/Graphs/Graph1.jsx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+import { Column } from "@ant-design/plots";
+
+import "./Graph1.css";
+
+function Graph1() {
+  const [data, setdata] = useState([]);
+
+  const GetApi = async () => {
+    let api =
+      "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY_ADJUSTED&symbol=IBM&outputsize=full&apikey=W9IPCZT6X76UJZOY";
```
Comment (Souvikdeb2612) [inline src/Components/Graphs/Graph1.jsx:12]: store url in state/variable

File: src/Components/Graphs/Graph1.jsx (hunk 2 above)
Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph1.jsx]: console statement is still present

File: src/Components/Graphs/Graph2.jsx (hunk 4 above)
Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph2.jsx]: It's still present

File: src/Components/Graphs/Graph2.jsx (hunk 5 above)
Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph2.jsx]: still present

File: src/Components/Graphs/Graph3.jsx (hunk 7 above)
Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph3.jsx]: still present

File: src/Components/Graphs/Graph3.jsx (hunk 8 above)
Comment (Souvikdeb2612) [reply src/Components/Graphs/Graph3.jsx]: still present

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**